test/functional/cmdLineTests/criu/src/org/openj9/criu/TestMultiThreadMode.java

```diff
@@ -0,0 +1,128 @@
+/*******************************************************************************
```

Comment:
do we have tests that confirm priority ordering is preserved?

Reply:
Added hookMode/priority tests.